libc/benchmarks/gpu/LibcGpuBenchmark.cpp

```diff
@@ -73,10 +74,16 @@ struct AtomicBenchmarkSums {
 };
 
 AtomicBenchmarkSums all_results;
+const char *header_format_string =
+    "Benchmark            |  Cycles |     Min |     Max | Iterations |   Time "
+    "(ns) |   Stddev |  Threads |\n";
```

Comment:
We should do what the unit tests do and scale the time value to the most readable unit. I.e. `1000 ms` vs. `1000000000 ns`.